app/src/main/java/pl/futurecollars/invoicing/service/JsonService.java

```diff
@@ -0,0 +1,35 @@
+package pl.futurecollars.invoicing.service;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import pl.futurecollars.invoicing.model.Invoice;
+
+public class JsonService {
+
+    private final ObjectMapper objectMapper;
+
+    {
```

Comment:
static block is fine, but in most case I don't like it - it is fine to just use no argument constructor